### DISCUSSION TITLE
How to adjust CPU/MEM requests and limits on OpenShift

### DISCUSSION THREAD

**Discussion by siwyroot** [fluxcd/flux2#3568] [Q&A]
Hello, thanks for a really nice software, helped me get hole cluster configuration using flux. I would like get more performance out of flux as we run multiple helm charts and quite a lot of manifests so we get OOMKilled or CPU throttling on controllers with default config. I can patch stuff manually but it's get overwritten with update. Is there a way to adjust CPU/MEM requests and limits config option at the time of installation using operator on OpenShift or after by some patch or something that could be applied by flux from git repo?

**Comment by stefanprodan** [comment]
If you have installed Flux from [OperatorHub](https://operatorhub.io/operator/flux) then you can't make any changes to it. I recommend you switch to [flux bootstrap](https://fluxcd.io/flux/installation/#bootstrap) as described in our [docs for OpenShift](https://fluxcd.io/flux/use-cases/openshift/#flux-installation-with-cli). Once you do that, then you can fine tune the Flux controllers as describe [here](https://fluxcd.io/flux/cheatsheets/bootstrap/).

The relevant configuration for your issue is here https://fluxcd.io/flux/cheatsheets/bootstrap/#increase-the-number-of-workers

**Comment by siwyroot** [comment]
That complicates few thing, could modification of requests, limits and controller config options could be added via additional config map?
